lib/moneta/adapters/memory.rb

```diff
@@ -7,6 +7,7 @@ class Memory
       include HashAdapter
       include IncrementSupport
       include CreateSupport
+      include EachKeySupport
```

Comment:
As the backend for this adapter is just a Ruby hash, I would have thought that in this case, `each_key` would be implemented by directly calling `each_key` on the underlying hash instead of using `EachKeySupport`.  Do you have a reason for not doing it this way?